src/modules/accumulate/process.cpp

```diff
@@ -22,11 +22,14 @@ EnumOptions<AccumulateModule::TargetPartialSet> AccumulateModule::targetPartialS
 }
 
 // Run main processing
-bool AccumulateModule::process(Dissolve &dissolve, const ProcessPool &procPool)
+enum Module::executionResult AccumulateModule::process(Dissolve &dissolve, const ProcessPool &procPool)
 {
     // Get the modules and decide on the PartialSet data name we're looking for
     if (targetModules_.empty())
-        return Messenger::error("No target modules set.");
+    {
+        Messenger::error("No target modules set.");
+        return failed;
```

Comment:
I don't recall whether using `enum class` will enforce this, but to me this makes the intent clearer:
```suggestion
        return ExecutionResult::Failed;
```